src/test/scala/beam/analysis/physsim/BeamCalcLinkStatsSpec.scala

```diff
@@ -26,8 +26,8 @@ class BeamCalcLinkStatsSpec extends WordSpecLike with Matchers with BeforeAndAft
 
   private var beamCalcLinkStats: BeamCalcLinkStats = _
 
-  private val TFHOURS = 31
-  private val TYPESTATS = 3
+  private val TFHOURS = 30
+  private val TYPESTATS = 1
```

Comment:
you should either use scala style or java style:
Scala style:
```Scala
TypeStats = 1
```

Java style:
```
TYPE_STATS = 1
```
```

Reply:
Hi @carloscaldas . I have change constants in scala style